.github/workflows/validator.yml

```
@@ -80,22 +78,22 @@ jobs:
     - name: Checking BIDS compliance
       if: always()
       run: |
-        bids-validator --verbose ./ | tee bids.txt
+        deno run --no-prompt --allow-env=BIDS_SCHEMA,FORCE_COLOR --allow-read=. jsr:@bids/validator . | tee bids.txt
```

I don't like a blanket `--allow-env`, especially in an environment like Github Actions which might contain secrets, but at least there's no network access granted so maybe it's ok.
```suggestion
        deno run --no-prompt --allow-env --allow-read=. jsr:@bids/validator . | tee bids.txt
```